let x = '';

try {

} catch (error) {

} finally {
    // 무조건 실행 : 정상 or 에러 모두
}